src/views/pacbio/PacbioRun.vue

```diff
@@ -6,6 +6,7 @@
       <b-button id="backToRunsButton" class="float-right">Back</b-button>
     </router-link>
 
+    <b-button v-if="currentAction.id == 'create'" variant="primary" class="float-right" @click="resetRun()" id="reset">Reset</b-button>
```

Comment:
Could use `this.newRecord` or `this.id === "new"` possbily instead of currentAction,